src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig

```diff
@@ -94,6 +94,18 @@
     {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/Modal/view_all_messages_modal.html.twig' %}
     {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/Modal/view_product_pack_modal.html.twig' %}
   </div>
+
+  <script type="text/html" id="modalTranslationsAddProductWithNewInvoice"
```

Comment:
It bothers me to create a script tag just to store these translations 🤔
Maybe it could be added on the select field instead? since it's related to the modal

Reply:
moved to selectbox